### PR TITLE
[libra framework] fix all TODOs in LibraBlock

### PR DESCRIPTION
**Pull request by sblackshear**
- Address TODO about making block prologue private. This changes the module bytecode, but is not a client-breaking change and need not be deployed to testnet.
- Remove TODO about triggering reconfiguration in block prologue. Reconfiguration should only be triggerable by LibraRoot or a validator operator
- Remove TODO about block height. We have decided to keep it.

### CODE REVIEW

File: language/stdlib/modules/LibraBlock.move [hunk 1]
```diff
@@ -90,13 +86,12 @@ module LibraBlock {
...
-                round: round,
-                proposer: proposer,
-                previous_block_votes: previous_block_votes,
+                round,
+                proposer,
+                previous_block_votes,
                 time_microseconds: timestamp,
             }
         );
-        // TODO(valerini): call regular reconfiguration here LibraSystem2::update_all_validator_info()
```
Comment (sblackshear) [inline language/stdlib/modules/LibraBlock.move:99]: @valerini: Just double-checking that this is an outdated comment left over from when we used to trigger reconfiguration automatically at block boundaries.

Comment (davidiw) [reply language/stdlib/modules/LibraBlock.move:99]: she may be offline for a bit, if memory serves me correct, this was intended to be used for batching within a block.

Comment (bob-wilson) [reply language/stdlib/modules/LibraBlock.move:99]: If she's not around, maybe file a GitHub issue about looking into this and then go ahead with the change. It's not like removing a comment is going to break anything....

Comment (sblackshear) [reply language/stdlib/modules/LibraBlock.move:99]: I remember the same as David, but I will double-check with Lera when she gets back.

**Pull request closed without merging**